test/utils/test-swagger-gen.ts

```diff
@@ -851,6 +852,16 @@ const testSwaggers: { [name: string]: SwaggerConfig } = {
     addCredentials: false,
     isTestPackage: true
   },
+  corecompattest: {

```

Comment:
Since we don't have any actual tests for this package, I think it would make sense to make this a SmokeTest instead. What do you think?

Reply:
Added the test